common/src/main/java/io/netty/util/internal/MacAddressUtil.java

```diff
@@ -110,27 +109,90 @@
             }
         }
 
-        if (bestMacAddr == NOT_FOUND) {
+        if (bestMacAddr == EMPTY_BYTES) {
             return null;
         }
 
         switch (bestMacAddr.length) {
             case 6: // EUI-48 - convert to EUI-64
-                byte[] newAddr = new byte[MAC_ADDRESS_LENGTH];
+                byte[] newAddr = new byte[EUI64_MAC_ADDRESS_LENGTH];
                 System.arraycopy(bestMacAddr, 0, newAddr, 0, 3);
                 newAddr[3] = (byte) 0xFF;
                 newAddr[4] = (byte) 0xFE;
                 System.arraycopy(bestMacAddr, 3, newAddr, 5, 3);
                 bestMacAddr = newAddr;
                 break;
             default: // Unknown
-                bestMacAddr = Arrays.copyOf(bestMacAddr, MAC_ADDRESS_LENGTH);
+                bestMacAddr = Arrays.copyOf(bestMacAddr, EUI64_MAC_ADDRESS_LENGTH);
         }
 
         return bestMacAddr;
     }
 
     /**
+     * Returns the result of {@link #bestAvailableMac()} if non-{@code null} otherwise returns a random EUI-64 MAC
+     * address.
+     * @return the result of {@link #bestAvailableMac()} if non-{@code null} otherwise returns a random EUI-64 MAC
```

Comment:
@Scottmitch just remove the `@return` as it is the same as what is stated already ?